trackpy/artificial.py

```diff
@@ -0,0 +1,180 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import six
+import numpy as np
+from scipy.spatial import cKDTree
+from trackpy.utils import validate_tuple
+
+
+def draw_point(image, pos, value):
+    image[tuple(pos)] = value
+
+
+def feat_gauss(r, rg=0.333):
+    """ Gaussian at r = 0 with max value of 1. Its radius of gyration is
+    given by rg. """
+    return np.exp((r/rg)**2 * r.ndim/-2)
```

Comment:
n [v0.2.4](https://github.com/soft-matter/trackpy/blob/41f800bcdaa403c5df4497e8690b974166857735/trackpy/tests/test_feature.py), the Gaussian is $e^{-(x^2+y^2)/(ndim*rg^2)}$
in which x, y are the meshgrids of coordinates and rg the absolute Rg.

If I write down the integrals for Rg of the general Gaussian $e^{-a r^2}$, I end up with $a = ndim / (2 *rg^2)$. So the factor in the v0.2 Gaussian is wrong. For 2D, there is a factor 2 difference, that I also mean in https://github.com/caspervdw/trackpy/commit/75877a4dd39b27253dc78afa6e1f85597ff48fec.